banyand/measure/query.go

```diff
@@ -393,6 +393,9 @@ func (qr *queryResult) Pull() *pbv1.MeasureResult {
 					offset := qr.tagNameIndex[tagProj]
 					tagFamilySpec := qr.schema.GetTagFamilies()[offset.FamilyOffset]
 					tagSpec := tagFamilySpec.GetTags()[offset.TagOffset]
+					if tagSpec.IndexedOnly {
+						continue
+					}
```

Comment:
It would be better for you to update write.go instead of query.go. "IndexOnly" means the tag is not stored in the file.